### ISSUE TITLE
PopupWidget to show with specified styles and specified place

### ISSUE THREAD

**Issue by arjunk** [tcampb/react-calendly#17]
Hi,
At the moment it seems PopupWidget can only be shown at the right bottom corner. 
However, on our webpage, we have Call to action at different places. 
Is there a way to show the Popup Widget at a particular place on the page, that is inside a particular HTML container?
Also with a particular style, let's say reducing the border-radius to make it consistent with the rest of the website theme. 

Because of the lack of these features, at the moment we tried PopupText. However, that also doesn't seem to have the ability to provide styling like text colour. This is the closest we could get: https://avniproject.org/

**Comment by tcampb** [comment]
Hi @arjunk, 

Thanks for the feedback! I'll look into adding additional style options to the `PopupWidget` component. In the meantime, you can add the following style tags to move the widget to the center or left side of the page:

```html
<!-- left align the badge widget -->
<style type="text/css"> 
.calendly-badge-widget { right: auto !important; left: 20px !important; } 
</style> 
<!-- end left align the badge widget -->
```

```html
<!-- center align the badge widget-->
<style type="text/css">
.calendly-badge-widget { left: 50% !important; margin-left: -100px!important; }
</style>
<!--end center align the badge widget-->
```

More information can be found here https://help.calendly.com/hc/en-us/articles/360019861794-Common-embed-questions#1

**Comment by arjunk** [comment]
Hi @tcampb,
Thanks so much for the prompt reply and registering the feedback. Looking forward to it being available. 
Meanwhile will give a try to what you suggested.

**Comment by tcampb** [comment]
Hi @arjunk,

I just published `v1.1.0` which includes the following changes:

* Added `styles` prop to the `PopupText` component so that the text color can be changed.
* Added a `openPopupWidget` function that can be used to create a custom Calendly button.

example:

```tsx
import { openPopupWidget } from 'react-calendly';

const CustomButton = ({ url, prefill, pageSettings, utm }) => {
  const onClick = () => openPopupWidget({ url, prefill, pageSettings, utm });

  return <button onClick={onClick}>Custom Button</button>
}
```

Hope this helps! Thanks again for your feedback!

**Comment by arjunk** [comment]
Hi @tcampb,
Thanks so much. That solved my problem. 
(styles prop to the PopupText was good enough. I haven't tried openPopupWidget).
